packages/synthetics-sdk-broken-links/src/link_utils.ts

```diff
@@ -58,8 +60,8 @@ export function checkStatusPassing(
  * @param options - The options object to be filled with default values.
  */
 export function setDefaultOptions(
-  options: BrokenLinksResultV1_BrokenLinkCheckerOptions
-) {
+  options: BrokenLinkCheckerOptions
```

Comment:
maybe consider renaming `options` to `input_options` to be consistent with `output_options` nomenclature + clarity